pyDeltaRCM/iteration_tools.py

```diff
@@ -42,9 +42,6 @@ def run_one_timestep(self):
         RuntimeError
             If model has already been finalized via :meth:`finalize`.
         """
-        if self._is_finalized:
-            raise RuntimeError('Cannot update model, model already finalized!')
-
```

Comment:
> By removing these lines, I don't think the Runtime Error will be thrown when this method is called anymore. 

Yes, that is correct. I will remove the note about that from the documentation in `run_one_timestep`, thanks. 

However, what I actually did here is *moved* this runtime error into the `update` method (which calls `run_one_timestep`). I think it makes more sense there, since 1) that is the more public-facing method, and 2) that's the method that actually calls the `output_data` method which it where it would be problematic to try and write to a finalized netcdf file. I guess that the error could instead (also) be raised there, but I think this makes sense too.

Reply:
That sounds good and makes sense to me